crates/precompile/Cargo.toml

```diff
@@ -21,7 +21,8 @@ secp256k1 = { version = "0.27.0", default-features = false, features = [
 ], optional = true }
 sha2 = { version = "0.10.5", default-features = false }
 sha3 = { version = "0.10.7", default-features = false }
-fhe_precompiles = { git = "https://github.com/Sunscreen-tech/fhe_precompiles", rev = "c3a2c184eb8ef7a136cc79f9e43567025d431be8" }
+# fhe_precompiles = { git = "https://github.com/Sunscreen-tech/fhe_precompiles", rev = "c3a2c184eb8ef7a136cc79f9e43567025d431be8" }
+fhe_precompiles = { path = "/Users/ryan/code/fhe_precompiles" }
```

Comment:
Just remember to point this toward GH before merging. Also FYI as long as the commit is pushed to the remote you could reference it here.

Reply:
What is kinda annoying is that the squash produces a new commit. Anyway, while working locally I haven't found a great solution unless I push up all my commits constantly to whatever PR I am testing against (if the PR/branch exists).